manual_correction.py

```diff
@@ -41,37 +38,104 @@ def get_parser():
+            - sub-001_T1w.nii.gz
+            - sub-002_T1w.nii.gz
             FILES_PMJ:
-            - sub-1000032_T1w.nii.gz
-            - sub-1000710_T1w.nii.gz\n
+            - sub-001_T1w.nii.gz
+            - sub-002_T1w.nii.gz\n
             """)
     )
     parser.add_argument(
         '-path-in',
         metavar="<folder>",
-        help='Path to the processed data. Example: ~/ukbiobank_results/data_processed',
-        default='./'
+        required=True,
+        help='Path to the processed data. Example: ~/<your_dataset>/data_processed',
     )
     parser.add_argument(
         '-path-out',
         metavar="<folder>",
-        help="Path to the BIDS dataset where the corrected labels will be generated. Note: if the derivatives/ folder "
-             "does not already exist, it will be created."
-             "Example: ~/data-ukbiobank",
+        help=
+        "R|Path to the output folder where the corrected labels will be saved. Example: ~/<your_dataset>/"
+        "Note: The path provided within this flag will be combined with the path provided within the "
+        "'-path-derivatives' flag. ",
         default='./'
     )
+    parser.add_argument(
+        '-path-derivatives',
+        metavar="<folder>",
+        help=
+        "R|Path to the 'derivatives' BIDS-complaint folder where the corrected labels will be saved. "
+        "Example: derivatives/labels"
+        "Note: if the provided folder (e.g., 'derivatives/labels') does not already exist, it will be created."
+        "Note: if segmentation or labels files already exist and you would like to correct them, provide path to them "
+        "within this flag.",
+        default=os.path.join('derivatives', 'labels')
+    )
+    parser.add_argument(
+        '-suffix-files-in',
+        help=
+        "R|Suffix of the input files. For example: '_RPI_r'."
+        "Note: this flag is useful in cases when the input files have been processed and thus contains a specific "
+        "suffix.",
+        default=''
+    )
+    parser.add_argument(
+        '-suffix-files-seg',
+        help="FILES-SEG suffix. Available options: '_seg' (default), '_label-SC_mask'.",
+        choices=['_seg', '_label-SC_mask'],
+        default='_seg'
+    )
+    parser.add_argument(
+        '-suffix-files-gmseg',
+        help="FILES-GMSEG suffix. Available options: '_gmseg' (default), '_label-GM_mask'.",
+        choices=['_gmseg', '_label-GM_mask'],
+        default='_gmseg'
+    )
+    parser.add_argument(
+        '-suffix-files-lesion',
+        help="FILES-LESION suffix. Available options: '_lesion' (default).",
+        choices=['_lesion'],
+        default='_lesion'
+    )
+    parser.add_argument(
+        '-suffix-files-label',
+        help="FILES-LABEL suffix. Available options: '_labels' (default), '_labels-disc'.",
+        choices=['_labels', '_labels-disc'],
+        default='_labels'
+    )
+    parser.add_argument(
+        '-label-list',
```

Comment:
```suggestion
        '-labels-list',
```

To be consistent with the suffix associated with disc labels (that is plural)

In https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/94, you don't put it `labels` but `label` should we change it for now just to be consistent?

Reply:
This argument is indented to allow passing a value or range of values for disc labeling. For example, `label-list 2:3` to label discs `2` and `3`.
Then, maybe, the argument should be named `-label-disc-list` to be compatible with the [`_label-disc.nii.gz` suffix](https://github.com/neuropoly/intranet.neuro.polymtl.ca/blob/jv/centralize_derivatives_convection/data/dataset-curation.md#derivatives-structure).

Reply:
Renamed in https://github.com/spinalcordtoolbox/manual-correction/pull/2/commits/f46935564edee0cccca92fc0c9df42e65ad88695